contracts/governance/src/wasm.rs

```diff
@@ -6,3 +6,60 @@
 
 #[cfg(not(feature = "no-bridge"))]
 mod bridge;
+
+use crate::{Error, GovernanceContract};
+use dusk_bls12_381_sign::PublicKey as BlsPublicKey;
+
+impl GovernanceContract {
+    /// Update the authority public address of the Governance Contract
+    pub fn update_authority(&mut self, key: BlsPublicKey) -> Result<(), Error> {
+        self.authority = key;
+
+        Ok(())
+    }
+    /// Update the broker public address of the Governance Contract
+    pub fn update_broker(&mut self, key: BlsPublicKey) -> Result<(), Error> {
+        self.broker = key;
+
+        Ok(())
+    }
```

Comment:
These methods should be called by rusk-recovery in order to set the values while generating the genesis state.
https://github.com/dusk-network/rusk/blob/55294da395d7f90bc3102541ffce517c1ea2e2f5/rusk-recovery/src/state.rs#L65-L72

Since this, I suggest to not having there into `wasm.rs` because the aforementioned is available only when compiling for wasm32 architecture

Reply:
@herr-seppia yea I misplaced this in `wasm.rs` fixing